recording/lib/recording_endpoint.ex

```diff
@@ -128,7 +147,7 @@ defmodule Membrane.RTC.Engine.Endpoint.Recording do
       |> Enum.filter(&Map.has_key?(ctx.children, &1))
 
     track_children = track_elements ++ track_sinks
-    {_track, state} = pop_in(state, [:tracks, track_id])
+    {_track, state} = pop_in(state, [:subscriptions_state, :tracks, track_id])
```

Comment:
As above I think we that `Subscribe` should remove it by invoking `remove_track/s`